backend/graphql-gateway/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ghcr.io/podkrepi-bg/go-devcontainer:v1.0.0 AS build
+WORKDIR /app
+COPY . /app
+
+RUN task build
+
+FROM alpine:3.14
```

Comment:
I would like to add some sensible defaults for the 2 env variables that this module currently has - `PORT` and `ACCOUNT_SERVICE_URL`. @kachar what should those be?

Reply:
This url should be the way graphql gateway contacts the grpc services like account, campaigns, etc

Probably it will be provided from the api gateway

Not sure there will be a default value for it as it depends on the stack setup, not on this individual component

Reply:
Then I will not mention it then and will just add it to the README